components/script/dom/webglframebuffer.rs

```diff
@@ -84,13 +84,36 @@ pub enum WebGLFramebufferAttachmentRoot {
     Texture(DomRoot<WebGLTexture>),
 }
 
+struct DroppableField {
+    id: WebGLFramebufferId,
+    is_deleted: Cell<bool>,
+    context: WebGLRenderingContext,
```

Comment:
We can do the same thing as the previous commit with a sender instead of a context.

Reply:
I fixed. https://github.com/servo/servo/pull/28302/commits/d77891ff6743928e56946e5ff42d4ff8a5360762